src/sedcli_main.c

```diff
@@ -329,10 +329,30 @@ static int get_password(struct sed_key *pwd)
 	return 0;
 }
 
+static void sed_status_ret(int status)
+{
+	const char *sed_status = NULL;
+
+	sed_status = sed_error_text(status);
+
+	if (sed_status == NULL && status != 0) {
+		if (status == -EINVAL) {
+			sedcli_printf(LOG_ERR, "Invalid parameter\n");
+		} else if (status == -ENODEV) {
+			sedcli_printf(LOG_ERR, "Couldn't determine device state\n");
+		} else if (status == -ENOMEM) {
+			sedcli_printf(LOG_ERR, "No memory\n");
+		} else {
+			sedcli_printf(LOG_ERR, "Unknown error\n");
+		}
+	 } else {
```

Comment:
please fix indentation

Reply:
This else is for the top most if 
also, I use the 'set list' feature on vim to check the indentation, it shows no issue

Reply:
> This else is for the top most if
> also, I use the 'set list' feature on vim to check the indentation, it shows no issue

there extra single whitespace before '}', and before 'sedcli_print...' and before '}'